test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file is meant to be ignored by git. kindly fix that 👍

Reply:
Hey 👋🏽  this is the last thing you need to handle 1️⃣  this file is supposed to be ignored by git, to fix that delete the file you have on Git, pull from this branch `feature`, and create this file again locally, this time it won't be added to git and your project will be approved. 🚀